<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify</class>
 <widget class="QDialog" name="Modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background:url(../../Images_projet/HMR.jpg) ;
}</string>
  </property>
  <widget class="QFrame" name="Modify_Accused">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>551</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background:#51556e;
    border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/QTprojet/smart court.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Mid">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lmid">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lmname">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mlastname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lmlastname">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="Macc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Accusation Cause</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="LmDate_B">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Mdb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Date Of Birth</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mgender">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mds">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>Session Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="LMDateS">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lmacc_cause">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Aa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LMgender">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose your Gender</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="modify">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:white;
border-radius: 15px;
background:#303030;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white
}</string>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
